shared/middleware/helpers/bucket_helper.rb

```diff
@@ -22,7 +22,12 @@ def list(encrypted_channel_id)
     @s3.list_objects(bucket: @bucket, prefix: prefix).contents.map do |fileinfo|
       filename = %r{#{prefix}(.+)$}.match(fileinfo.key)[1]
       mime_type = Sinatra::Base.mime_type(File.extname(filename))
-      category = mime_type.split('/').first  # e.g. 'image' or 'audio'
+
+      if mime_type == 'application/pdf'
+        category = 'pdf'
+      else
+        category = mime_type.split('/').first  # e.g. 'image' or 'audio'
+      end
```

Comment:
Any good place to put a `category_from_mime_type` helper, since we do this twice and it might keep growing?